test/verifyImageLabels.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+VCS_REF=$(docker inspect -f '{{ index .Config.Labels "org.label-schema.vcs-ref"}}' wurstmeister/kafka)
+echo "VCS_REF=$VCS_REF"
+if [ -z "$VCS_REF" ] || [ "$VCS_REF" = "unspecified" ]; then
```

Comment:
This test could be improved to check that the sha is a valid commit in the master git tree.